src/CSSBattle/3_PushButton/Yongbin.html

```diff
@@ -0,0 +1,47 @@
+<div class="rectangle">
```

Comment:
Since you are using PascalCase here, consider this renaming
```suggestion
<div class="Rectangle">
```

Reply:
Resolved in commit 5ec733b.